releases/net471/KnownIssues/623552-BCL Higher assembly versions that 4.0.0.0 for System.IO.Compression.ZipFile cannot be loaded without a binding redirect.md

```diff
@@ -0,0 +1,36 @@
+# Higher assembly versions that 4.0.0.0 for System.IO.Compression.ZipFile cannot be loaded without a binding redirect
```

Comment:
Better: Assembly versions higher than 4.0.0.0 for System.IO.Compression.ZipFile...